finality-aleph/src/network/manager/service.rs

```diff
@@ -585,7 +585,7 @@ impl<GE: Display> Display for Error<GE> {
         match self {
             CommandsChannel => write!(f, "commands channel unexpectedly closed"),
             MessageChannel => write!(f, "message channel unexpectedly closed"),
-            ValidatorNetwork => write!(f, "validator network unexpectedly done"),
+            CliqueNetwork => write!(f, "validator network unexpectedly done"),
```

Comment:
```suggestion
            CliqueNetwork => write!(f, "clique network unexpectedly done"),
```

Reply:
Actually I did the opposite, for the same reason as I kept the variable names before. So now the error is called `ValidatorNetwork` again.